src/spaces/openstreetmap.jl

```diff
@@ -951,6 +951,26 @@ function Agents.nearby_ids(
     return nearby
 end
 
+function Agents.nearby_ids(
+        pos::Tuple{Float32, Float32},
+        model::ABM{<:OpenStreetMapSpace},
+        distance::Real,
+        args...;
+        kwargs...
+    )
+    nearby = []
```

Comment:
This is type unstable. It should be `Int[]`.

Reply:
Related: https://github.com/JuliaDynamics/Agents.jl/pull/934#discussion_r1403801703

Reply:
Type instability is fixed.